examples/user_guide/Django_Apps.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As the core user guides including the [Introduction](Introduction.ipynb) have demonstrated, it is easy to display Panel apps in the notebook, launch them from an interactive Python prompt, and deploy them as a standalone Bokeh server app from the commandline. However, it is also often useful to embed a Panel app in large web application, such as a Django web server. Using Panel with Django requires a bit more work than for notebooks and Bokeh servers.\n",
+    "As the core user guides including the [Introduction](../getting_started/index.html) have demonstrated, it is easy to display Panel apps in the notebook, launch them from an interactive Python prompt, and deploy them as a standalone Bokeh server app from the commandline. However, it is also often useful to embed a Panel app in large web application, such as a Django web server. Using Panel with Django requires a bit more work than for notebooks and Bokeh servers.\n",
```

Comment:
Shouldn't this have been ../getting_started/Introduction.ipynb?

Reply:
(That way the link should work both in Jupyter and on the website. Otherwise, it will fail in Jupyter because the .html version is not present in a user's copy of the notebooks.)

Reply:
There is a good chance that the error in my docs build setup that I referred to [here](https://github.com/pyviz/panel/issues/628#issuecomment-541533410) lead me to believe that I can only specify the index and not the notebook. I can push a fix, but I can't test it. It would be just exchanging the link for `../getting_started/Introduction.ipynb`

Reply:
@jbednar #709

Reply:
I should have reviewed with more care sorry.

Reply:
Actually I'm not entirely sure there is a consistent way to link this since Introduction.ipynb ends up being built as part of index.html.